internal/resources/fabric/route_filter_rule/resource_test.go

```diff
@@ -0,0 +1,97 @@
+package route_filter_rule_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/equinix/terraform-provider-equinix/internal/acceptance"
+	"github.com/equinix/terraform-provider-equinix/internal/resources/fabric/route_filter_rule"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+)
+
+func TestAccFabricRouteFilterRule_PFCR(t *testing.T) {
```

Comment:
The Fabric e2e tests seem to fail consistently in CI, which means we can't tell from here if things break.  Does your team have a way to run them successfully outside of CI (and/or a plan to get them running successfully in CI)?

Reply:
We do run them locally to verify additions during development. We've identified that a big reason why the tests fail is that we have overlapping user data between them.

An epic is In Progress for resolving all testing issues. It's just lagging behind feature development work.